internal/app/singularity/plugin_compile_linux.go

```diff
@@ -62,11 +64,11 @@ func CompilePlugin(sourceDir, destSif, buildTags string) error {
 	// build plugin object using go build
 	_, err := buildPlugin(sourceDir, buildTags)
 	if err != nil {
-		return fmt.Errorf("while building plugin .so: %s", err)
+		return fmt.Errorf("while building plugin .so: %v", err)
```

Comment:
Why change this to `%v`?

Reply:
Here it will not change the output, a matter of habit rather.